docs/advanced.md

```diff
@@ -179,7 +179,7 @@ cloud-controller-manager
 Thus, an example `--control-plane-resource-requests` or `--control-plane-resource-limits` value may look like:
 
 ```
-kube-apiserver-cpu=500m,kube-apiserver-memory=512MiB,kube-scheduler-cpu=250m,kube-scheduler-memory=512MiB,etcd-cpu=1000m
+kube-apiserver-cpu=500m,kube-apiserver-memory=512M,kube-scheduler-cpu=250m,kube-scheduler-memory=512M,etcd-cpu=1000m
```

Comment:
Either one is fine: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory